app/src/main/java/com/ivanpodlesnykh/playlistmaker/PlayerActivity.kt

```diff
@@ -13,30 +17,79 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 
 class PlayerActivity : AppCompatActivity() {
+
+    private lateinit var backButton: ImageView
+
+    private lateinit var playButton: ImageView
+
+    private lateinit var currentTrackTime : TextView
+
+    private lateinit var url: String
+
+    private var mainThreadHandler: Handler? = null
+
+    private var mediaPlayer = MediaPlayer()
```

Comment:
Сам плеер может быть `val`, т.к. он не пересоздаётся
```suggestion
    private val mediaPlayer = MediaPlayer()
```